src/components/VisualizationStepViews.tsx

```diff
@@ -81,7 +81,8 @@ const VisualizationStepViews = ({
   };
 
   useEffect(() => {
-    setActiveTabKey(configTabIndex);
+    setActiveTabKey(()=>
+    step.parameters?.length?configTabIndex:detailsTabIndex);
```

Comment:
Hey @pnaik1 thanks for your contribution! Do you mind to add a space here so it's easier to read? For example like this:
`step.parameters?.length ? configTabIndex : detailsTabIndex);`